airbyte-ci/connectors/metadata_service/lib/metadata_service/models/src/ConnectorMetadataDefinitionV0.yaml

```diff
@@ -25,7 +25,7 @@ properties:
       - githubIssueLabel
       - connectorSubtype
       - releaseStage
-    additionalProperties: false
+    additionalProperties: true
```

Comment:
Why are we allowing additionalProperties now?

Reply:
Is it to allow `_` prefixes? I'm not sure I get why we want to continue allowing `_` prefix and I'm afraid enabling `additionalProperties` weakens the validation and can lead to sneaky errors.

Reply:
So Were in a messy middle after discovering the pydantic model generated did not like the leading underscore.

The immediate fix(s) we deployed
1. Allowed prerealease to use the alias when writing the prerelease metadata file
1. Updated the additionalProperties to true for the registry to handle a parsing error resulting from above

Now going forward we've got a few metadata files out there that have _ab_internal when they should have ab_internal
This is why im setting additionalProperties to true, just so things dont blow up when rolling these new fields out, and in the next version of the metadata we can revert it back to false